delta/plugins/storage/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/StorageScopeInitializationSpec.scala

```diff
@@ -82,5 +82,13 @@ class StorageScopeInitializationSpec
       val resource = storages.fetch(nxv + "diskStorageDefault", project.ref).accepted
       resource.rev shouldEqual 1L
     }
+
+    "be instantiated when enabled" in {
+      StorageScopeInitialization.when(enabled = true)(storages, sa, defaults) shouldBe defined
```

Comment:
I think unless you can test our distage somehow, then in order to test this properly we would need to refactor how scopes are created. 

My point here is that even if this function works correctly, this test doesn't ensure that this is the method that gets called